main.js

```diff
@@ -1 +1,91 @@
+/* Objectives of this assignment:
 
+ -  understand how to use JS functions
+*/
+function functionName () {
+    console.log("");
+} 
+functionName
+
+// understand how to use JS algorithms
+
+for (function functionName "");
+ 
```

Comment:
You probably wouldn't put a function inside the () with a for loop. What are you looping over here?